### PR TITLE
Document Folder UI Tweaks

### PR DESCRIPTION
**Pull request by djnunez-aot**
*Issue #1027 :*
https://github.com/bcgov/met-public/issues/1027

-Change button order

-edit button styling

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

File: met-web/src/components/engagement/form/EngagementWidgets/Documents/CreateFolderForm.tsx [hunk 1]
```diff
@@ -77,10 +77,32 @@ const CreateFolderForm = () => {
                 sx={{ marginBottom: '3em' }}
             >
                 <Grid item>
-                    <SecondaryButton onClick={() => setCreateFolderMode(true)}>Create Folder</SecondaryButton>
+                    <SecondaryButton
```
Comment (jadmsaadaot) [inline met-web/src/components/engagement/form/EngagementWidgets/Documents/CreateFolderForm.tsx:80]: please create a new button in commons with this style so we can reuse it later.

Comment (djnunez-aot) [reply met-web/src/components/engagement/form/EngagementWidgets/Documents/CreateFolderForm.tsx:80]: sounds good

File: met-web/src/components/common/index.tsx [hunk 2]
```diff
@@ -47,6 +47,25 @@ export const StyledSecondaryButton = styled(MuiButton)(() => ({
     },
 }));
 
+export const StyledNeutralButton = styled(MuiButton)(() => ({
```
Comment (jadmsaadaot) [inline met-web/src/components/common/index.tsx:50]: I read in #1027 that MA called them WidgetButtons. Our naming basically defines how we are going to use them. So maybe we can call them WidgetButtons, a quick check with MA would be great about where this style is used

**Review by tiagograf-AOT — APPROVED** [review]

File: met-web/src/components/common/index.tsx (hunk 2 above)
Comment (djnunez-aot) [reply met-web/src/components/common/index.tsx]: Sure thing ill see what she says

**Review by jadmsaadaot — APPROVED** [review]

**Pull request merged**